element-template-generator/core/src/main/java/io/camunda/connector/generator/dsl/Property.java

```diff
@@ -40,6 +40,9 @@ public abstract sealed class Property
 
   protected final String type;
 
+  /**
+   * @Pavel
```

Comment:
?

Reply:
![image](https://github.com/user-attachments/assets/a5377d65-139c-4629-81b3-04a0bd883d8d)

FYI @sbuettner @chillleader @johnBgood 

I did not understand why Simon was asking me this (the `'s` was missing). I realize now how stupid I have been 🤣🤣